### PR TITLE
docs: add missing syntax highlight

### PR DESCRIPTION
**Pull request by Naturalclar**
## Description

Filling in some missing syntax highlighting for code snippets in the document.

### CODE REVIEW

File: docs/docs/using-graphql-playground.md [hunk 1]
```diff
@@ -20,13 +20,13 @@ To access this experimental feature utilizing GraphQL Playground with Gatsby, ad
 
 If you are on **Windows** then you should use:
 
-```
+```json
```
Comment (laurieontech) [inline docs/docs/using-graphql-playground.md]:
```suggestion
```json:title=package.json
```

**Comment by Naturalclar** [comment]
@laurieontech thanks for the suggestion! I didn't know that you could add titles to the snippets, I learn something new everyday!
I've also added a title in other json of this example.

**Comment by laurieontech** [comment]
It's great! I don't believe it's supported out of the box in markdown, it's part of our docs implementation. Thanks for the addition :)

**Comment by laurieontech** [comment]
It's great! I don't believe it's supported out of the box in markdown, it's part of our docs implementation. Thanks for the addition :)

**Pull request merged**